src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+db.driver=org.postgresql.Driver
+db.url=jdbc:postgresql://elmer.db.elephantsql.com:5432/mosveawi
+db.username=mosveawi
+db.password=TJATZtVnWyaYxY3e5yU2ZA1qGdDZh-_f
+hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+hibernate.show_sql=false
+hibernate.format_sql=true
+hibernate.hbm2ddl.auto=create-drop
```

Comment:
For me it can be merged. Nice to see password to your database :laughing:. The most important is to cut empty lines :+1: